src/main/resources/view/StatTabContent.fxml

```diff
@@ -15,9 +18,27 @@
             <Label text="Inventory" styleClass="list-header"/>
             <StackPane fx:id="ingredientListPanelPlaceholder" VBox.vgrow="ALWAYS"/>
         </VBox>
+
+        <VBox styleClass="pane-with-border" GridPane.columnIndex="1" GridPane.vgrow="ALWAYS">
+            <padding>
+                <Insets top="10" right="10" bottom="10" left="10" />
+            </padding>
+
+            <Label text="Business Analytics" styleClass="list-header"/>
```

Comment:
If our tab name and user guide calls this statistics, is it better to preserve the name?
Btw can you update the tab name and UiTab from `Inventory` to `Inventory & Statistics` thanks
```suggestion
            <Label text="Statistics" styleClass="list-header"/>
```